README.md

```diff
@@ -115,8 +115,14 @@ return {
         end,
       },
       keymaps = {
+        -- if false, no bindings will be provided at all
+        -- thus you will have to bind on your own
         use_default = true,
         -- key maps on the vuffers list
+        -- - may map multiple keys for the same action
+        --    open = { "<CR>", "<C-l>" }
```

Comment:
Very nice. Could you update `KeymapView` in `lua/vuffers/config.lua`? 

Now it's `@field <action_name> string` as in `@field open string`, but it now should be `@field <action_name> string | string[]`.